### ISSUE TITLE
Schedules are not running in latest Panel

### ISSUE THREAD

**Issue by ProJaCore** [Pterodactyl/Panel#882]
- Crontab is running and working (other commands..)
- pteroq.service is running
- schedules are getting updated "Next Run xx..."
## But Schedules are not getting executed (may be internal error)
I already tried it manually but it's still not working:
```
php artisan schedule:run
php artisan p:schedule:process
```
And there are also no logs.

---

In MYSQL it only updates the `schedules` table:
---
![grafik](https://user-images.githubusercontent.com/12472791/35071950-ece2658a-fbe2-11e7-90bf-fbd6a315abf8.png)

## but it does not touch the `jobs` table:
![grafik](https://user-images.githubusercontent.com/12472791/35071976-133ec228-fbe3-11e7-8dde-d39f6610bdfb.png)

---
* Version of Panel: bd238c75
* OS: Ubuntu 16.04
* Your Computer's OS & Browser: Win10 FFDE

**Comment by DaneEveritt** [comment]
The jobs table will only have things in it while there is something queued to run but not yet run. It will also only have things in it if you have the queue worker set to database.

The correct way to test this is disabling the pteroq service and making sure jobs are configured to store in the DB and seeing if any get placed there. I also need you to check the queue log and see if anything is in there relating to the queued jobs.

I just tested this the other day and it was working perfectly fine.

**Comment by ProJaCore** [comment]
here is the syslog https://hastebin.com/ikaluyodaw.rb

**Comment by DaneEveritt** [comment]
Please provide errors from the failed jobs table. That log shows then running correctly.

**Comment by ProJaCore** [comment]
There are no other logs

**Comment by DaneEveritt** [comment]
Is there anything in the failed_jobs table?

**Comment by ProJaCore** [comment]
@DaneEveritt 

```
{"maxTries":null,"displayName":"Pterodactyl\\Jobs\\Schedule\\RunTaskJob","timeout":null,"job":"Illuminate\\Queue\\CallQueuedHandler@call","id":"giiPQ2tvojtjmFV4u6cgkeBiUvpXTrSS","timeoutAt":null,"data":{"command":"O:36:\"Pterodactyl\\Jobs\\Schedule\\RunTaskJob\":12:{s:20:\"\u0000*\u0000commandRepository\";N;s:18:\"\u0000*\u0000powerRepository\";N;s:8:\"schedule\";i:4;s:4:\"task\";i:7;s:17:\"\u0000*\u0000taskRepository\";N;s:10:\"connection\";N;s:5:\"queue\";s:8:\"standard\";s:15:\"chainConnection\";N;s:10:\"chainQueue\";N;s:5:\"delay\";i:0;s:7:\"chained\";a:0:{}s:6:\"\u0000*\u0000job\";N;}","commandName":"Pterodactyl\\Jobs\\Schedule\\RunTaskJob"},"attempts":2}
```

**Comment by DaneEveritt** [comment]
Sorry, the exception column from that table :)

**Comment by ProJaCore** [comment]
Oh yeah, Sorry but i run a Distinct sql query to get all different Exceptions so yeeah:
https://1drv.ms/x/s!AsOCnNfThtbqgRA69cCqYskxrdWw

**Comment by DaneEveritt** [comment]
And theres no errors on the daemon while this is happening? Your schedules are running just fine, they're just hitting an error when sending data to the daemon which is why you're not seeing anything in your server console.

**Comment by ProJaCore** [comment]
After a Daemon  restart, this error came up:
`{"name":"wings","hostname":"srv01","pid":30728,"level":60,"err":{"message":"(HTTP code 500) server error - unauthorized: access to the requested resource is not authorized ","name":"Error","stack":"Error: (HTTP code 500) server error - unauthorized: access to the requested resource is not authorized \n    at /srv/daemon/node_modules/docker-modem/lib/modem.js:254:17\n    at IncomingMessage.<anonymous> (/srv/daemon/node_modules/docker-modem/lib/modem.js:281:9)\n    at emitNone (events.js:91:20)\n    at IncomingMessage.emit (events.js:185:7)\n    at endReadableNT (_stream_readable.js:974:12)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickDomainCallback (internal/process/next_tick.js:128:9)"},"additional":{"reason":"server error","statusCode":500,"json":null},"msg":"A fatal error caused the daemon to abort the startup.","time":"2018-01-20T20:41:31.160Z","v":0}`

**Comment by DaneEveritt** [comment]
Is this issue being caused by your daemon issues or is it something else?

**Comment by ProJaCore** [comment]
@DaneEveritt its caused because of the old docker image. 
IMPORTANT: I cant delete the server with the old docker image. even if I did forcibly delete it, it did not solve the problem in the daemon. the daemon is still trying to download the old image for the given server. that's why it's a bad practice to make the daemon not independent from the associated game servers. 

The daemon should be able to start independently from the content because its the only connection between panel and node. You're not able to delete servers or do changes when the daemon fails to start. 
I also don't understand why I should delete the server if I did just change the location of the image or if I can just change the image.
